systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java

```diff
@@ -57,9 +57,9 @@
 import static io.strimzi.systemtest.k8s.Events.Unhealthy;
 import static io.strimzi.systemtest.matchers.Matchers.hasAllOfReasons;
 import static io.strimzi.systemtest.matchers.Matchers.hasNoneOfReasons;
-import static io.strimzi.test.StrimziExtension.ACCEPTANCE;
-import static io.strimzi.test.StrimziExtension.REGRESSION;
-import static io.strimzi.test.StrimziExtension.TOPIC_CM;
+import static io.strimzi.test.extensions.StrimziExtension.ACCEPTANCE;
+import static io.strimzi.test.extensions.StrimziExtension.REGRESSION;
+import static io.strimzi.test.extensions.StrimziExtension.TOPIC_CM;
```

Comment:
This is an aside but seeing this makes we wonder whether there's any value in having a tag for flaky tests. That way it would be easy to test just the flaky ones (when trying to improve them) and to avoid running the flaky ones in some circumstances. It could thus be a better way of including/excluding tests than changing the tags we already have.

Reply:
Ye that's a good point since we have few flaky tests now. I will create new PR with new tag and changes for tests.